File: ShadowFrameworkLite20/src/shadow/renderer/data/SFReferenceNodeData.java
package shadow.renderer.data;

import shadow.renderer.SFAbstractReferenceNode;
import shadow.renderer.SFReferenceNode;

public class SFReferenceNodeData extends SfAbstractReferenceNodeData {

	@Override
	protected SFAbstractReferenceNode generateReferenceNode() {
		return new SFReferenceNode();
	}
	
}

File: ShadowFrameworkJS/src/shadow/renderer/data/SFReferenceNodeData.js


function SFReferenceNodeData(){
	this.generate();
}

inherit(SFReferenceNodeData,SFAbstractReferenceNodeData);

SFReferenceNodeData.prototype["generateReferenceNode"]=function(){
	return new SFReferenceNode();
};


SFReferenceNodeData.prototype["generateNewDatasetInstance"]=function(){
	return new SFReferenceNodeData();
};